books/RayTracingInOneWeekend.html

```diff
@@ -386,19 +386,20 @@
 --------------
 <div class='together'>
 The one thing that all ray tracers have is a ray class, and a computation of what color is seen
-along a ray. Let’s think of a ray as a function $\mathbf{p}(t) = \mathbf{a} + t \vec{\mathbf{b}}$.
-Here $\mathbf{p}$ is a 3D position along a line in 3D. $\mathbf{a}$ is the ray origin and
-$\vec{\mathbf{b}}$ is the ray direction. The ray parameter $t$ is a real number (`double` in the
-code). Plug in a different $t$ and $p(t)$ moves the point along the ray. Add in negative $t$ and you
-can go anywhere on the 3D line. For positive $t$, you get only the parts in front of $\mathbf{a}$,
-and this is what is often called a half-line or ray.
+along a ray. Let’s think of a ray as a function $\mathbf{P}(t) = \mathbf{A} + t \mathbf{b}$. (Note:
+throughout these books, we'll use uppercase bold letters for points, and lowercase bold letters for
+vectors.) Here $\mathbf{P}$ is a 3D position along a line in 3D. $\mathbf{A}$ is the ray origin and
```

Comment:
I'm not sure if this note is necessary. It's a level of formalism that I question for the series

Reply:
Yeah, I'll pull it.